service/api_logic/filter_manager/games_filter_manager.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy.orm import Query
+from database.models import Games, Country, League
+from service.api_logic.scripts import get_sport_by_name
+from service.api_logic.filter_manager.base_filter_manager import BaseFilterManager
+from sqlalchemy import cast
+from sqlalchemy.sql.sqltypes import Time, Date
+
+class GamesFilterManager(BaseFilterManager):
+
+    @staticmethod
+    def apply_sport_filter(query: Query, value: int) -> Query:
```

Comment:
since filter has something to do with id we'd better reflect that in the name of the method